all-unix-style.yml

```diff
@@ -150,6 +150,22 @@ properties:
           [the github project](https://github.com/taskcluster/taskcluster-proxy) for more information.
 
           Since: generic-worker 10.6.0
+  onExitStatus:
+    title: Exit status handling
+    description: By default generic-worker will fail a task with a non-zero exit status
+      without retrying.  This payload property allows a task owner to define certain
+      exit statuses that will be marked as a retriable exception.
+    type: object
+    additionalProperties: false
+    properties:
+      retry:
```

Comment:
`exitCodes:` rather than `retry:`

Reply:
Note for others reading this, we think we're going to land this with a clone of docker-worker's behavior and then roll out this nicer form later.